Initial commit.

yakivmospan opened the pull request:
Changed launch icon for android.
Added splash screen for android.
Added basic theme.
Added basic home screen with basic navigation bar.

yakivmospan committed:
Initial commit.

Changed launch icon for android.
Added splash screen for android.
Added basic theme.
Added basic home screen with basic navigation bar.

ababicheva commented on README.md:
```diff
@@ -1,2 +1,13 @@
-# coffee-shop
+# CoffeeShop
 Flutter based sample application.
+
```
Let's update README with some project details, e.g.:
- what is coffee shop app for?
- technologies that we are using
- code style/conventions

yakivmospan replied:
Yep, can be updated with some more stuff, thanks

ababicheva replied:
please fix typos

yakivmospan replied:
@budgieParrot updated

ababicheva commented on android/app/build.gradle:
```diff
@@ -0,0 +1,67 @@
+
+    defaultConfig {
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+        applicationId "com.example.flutterapp"
```
let's use more company and project-specific app id

yakivmospan replied:
👍

ababicheva commented on android/app/build.gradle:
```diff
@@ -0,0 +1,67 @@
+    defaultConfig {
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+        applicationId "com.example.flutterapp"
+        minSdkVersion 16
```
should we support from 16th api? let's finally start at least from 21st!

yakivmospan replied:
This was autogenerated by flatter, I believe this is min that they support. I would like to keep it as it for the beginning and if needed change in future

https://github.com/flutter/flutter/issues/9310

yakivmospan replied:
https://flutter.dev/docs/resources/faq#what-devices-and-os-versions-does-flutter-run-on

ababicheva commented on android/app/src/debug/AndroidManifest.xml:
```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.flutterapp">
```
why a file from debug is in repo?

yakivmospan replied:
This was autogenerated by flatter.

yakivmospan replied:
Use debug mode during development, when you want to use hot reload.
https://flutter.dev/docs/testing/build-modes

ababicheva commented on android/app/src/main/AndroidManifest.xml:
```diff
@@ -0,0 +1,30 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.flutterapp">
```
package name - same as for app id

yakivmospan replied:
Yes, also need to update ios package as well

ababicheva commented on android/app/src/profile/AndroidManifest.xml:
```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.flutterapp">
```
I didn't read getting started guides for flutter. what is profile folder for?

yakivmospan replied:
This was autogenerated by flatter.

>Use profile mode when you want to analyze performance.
https://flutter.dev/docs/testing/build-modes

oleksandrakovalova commented on ios/Flutter/flutter_export_environment.sh:
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```
Add this file to .gitignore

yakivmospan replied:
@oleksandrakovalova updated

yakivmospan committed:
Initial commit.

Updated readme.
Updated package and project id.

yakivmospan committed:
Removed duplicated lines in readme

yakivmospan committed:
Initial setup:

Fixed typo's in readme
Updated git ignore rules, added auto generated files there
